sonar-scanner-engine/src/main/java/org/sonar/scanner/repository/ServerSideProjectDataLoader.java

```diff
@@ -19,7 +19,6 @@
  */
 package org.sonar.scanner.repository;
 
-
-public interface ProjectRepositoriesLoader {
-  ProjectRepositories load(String projectKeyWithBranch, boolean issuesMode);
+public interface ServerSideProjectDataLoader {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "ServerSideProjectDataLoader" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1609)